### PR TITLE
answers for lesson 1

### PR DESCRIPTION
**Pull request by frrf**

### CODE REVIEW

File: 01-node-tutorial/answers/04-names.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+//local
+const secret = 'SUPER SECRET'
+//shared
+const john = 'john'
+const peter = 'peter'
```
Comment (jdevries3133) [inline 01-node-tutorial/answers/04-names.js:5]: Challenge question: what's the difference between local and shared? How would we need to change this file to make `secret` shared or public?

File: 01-node-tutorial/answers/12-http.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  } else if (req.url === '/about') {
+    res.end('Here is our short history')
+  } else {
+    res.end(`
+    <h1>Oops!</h1>
+    <p>We can't seem to find the page you are looking for</p>
+    <a href="/">back home</a>
+    `)
+  }
+})
```
Comment (jdevries3133) [inline 01-node-tutorial/answers/12-http.js:15]: (optional) extra challenge: can you add a page to this "website"

**Review by jdevries3133 — APPROVED** [review]
Everything looks good! I'm on mobile now so I cannot test; let me know if you have any issues or questions.

File: 01-node-tutorial/answers/04-names.js (hunk 1 above)
Comment (frrf) [reply 01-node-tutorial/answers/04-names.js:5]: We would have to include the `secret` variable in the `exports` node module 

`module.exports = {john,peter,secret}`

Comment (jdevries3133) [reply 01-node-tutorial/answers/04-names.js:5]: Good, thanks!

File: 01-node-tutorial/answers/12-http.js (hunk 2 above)
Comment (frrf) [reply 01-node-tutorial/answers/12-http.js]: added a contact page :) 

https://github.com/Code-the-Dream-School/node-express-course/pull/24/commits/bac04824524cf9c18d540e3d1fc6746d802a9cbf

**Comment by jdevries3133** [comment]
Nice work!

File: 01-node-tutorial/answers/12-http.js (hunk 2 above)
Comment (itzjitesh) [reply 01-node-tutorial/answers/12-http.js]: > (optional) extra challenge: can you add a page to this "website"
const server = http.createServer((req, res) => {
  if (req.url === '/') {
    res.end('Welcome to our home page')
  } else if (req.url === '/about') {
    res.end('Here is our short history')
  } else if (req.url === '/services') {
    res.end('All the services are well maintained!')
  } else {
    res.end(`
    <h1>Oops!</h1>
    <p>We can't seem to find the page you are looking for</p>
    <a href="/">back home</a>
    `)
  }
})

Comment (jdevries3133) [reply 01-node-tutorial/answers/12-http.js]: Looks good, nice job!

**Pull request closed without merging**